FIAT/hellan_herrmann_johnson.py

```diff
@@ -66,20 +66,27 @@ def __init__(self, ref_el, degree, variant, qdegree):
             nodes.extend(PointwiseInnerProductEvaluation(ref_el, n[i+1], n[i+2], pt)
```

Comment:
Do we have proper FIAT-level tests for these that will ensure you're not borking anything in the elements, or are we going to hope for good code coverage in Firedrake?

Reply:
We have regression tests in FIAT for the non-macro elements. https://github.com/firedrakeproject/fiat/blob/5581c4e59be48515cf8458f7d2120f4a2984b10c/test/FIAT/regression/test_regression.py#L267

This PR is adding unisolvence tests for every macroelement we support (only for the lowest degree that gives interior DOFs). We also test correctness of the tiled `MacroPolynomialSet` (only the lowest order)

Reply:
Any chance to check correctness in finat?  But if we have unisolvence and separately check correctness of macro duals and macro polynomial sets, by composability nothing can go wrong, right?

Reply:
It is a bit strange to add a test just for the DualSet, we don't have those. I already put test transformation theory tests on macro BDM and Nedelec.

To test a macroelement, we can instantiate an element on a split cell and restrict it onto the indices of the closure of a subcell. The resticted element should be equal to an element created on the subcell.